flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg

```diff
@@ -17,10 +17,11 @@ major = 1
 minor = 0
 
 [WEIGHT_AND_BALANCE]
-max_gross_weight = 174165 ; Empty weight, (LBS)
-empty_weight = 90400 ; Empty weight, (LBS)
+max_gross_weight = 174165 ; Maximum total weight of the aircraft when fully loaded, (LBS)
+empty_weight = 97003 ; Empty weight, (LBS); Includes weight to account for pantry, galleys, lavs, potable water, emergency equipment etc.
```

Comment:
@donstim continuing from the #navigraph channel on Discord.

Have updated the Operating Empty weight to account for operator items which was marked as a TBD previously.

Reply:
As this was a TBD item, I would be more comfortable if @ghosh9691 were to review it first. Is he still around?

If he isn't, I suggest going with the actual TAP OEW (46, 452 kg) that Iceman found for an airplane with the same seating capacity rather than the 44,000 kg from Derek's comment saying "most sources I can find show it to be typically in the 44000 range."

Reply:
Makes sense, I shall DM him as well regarding this for a quicker response